lib/customer.rb

```diff
@@ -0,0 +1,39 @@
+require '../config'
+
+class Customer
+
+  def initialize(phone = nil)
+    @phone = phone
+  end
+
+  def input_number
+    puts "What number should we text a confirmation to?"
+    @phone = "+44#{gets.chomp}"
```

Comment:
I really like this! Small pedantic point, if someone enters their number as 07955... then you'll need to remove the 0 as well as add the +44